### PR TITLE
Nesting bleeding classes into overarching class

### LINKED ISSUE #1154: Add nesting to page card component in _sitemap.scss
### Overview
We need to nest the page-specific properties of the .pg-card-lg class in the site map page so that they don't affect page cards on other pages.

### Action Items
- [x] Nest the scss properties below (bottom of this issue) to make sure they will only apply to the page cards on the site map page

### Resources/Instructions

These properties should be nested (perhaps under `.sitemap-page` ? ) because they are affecting the page cards on the new Join Us page.

```
  @media #{$bp-tablet-up}{
.page-card-lg:nth-child(1n+2){
      padding-left: 0;
  }
```

(*This is my fault because I am supposed tocreate a generic _page-cards.scss file that all pages on the site can pull from, but I haven't done this yet due to a few dependencies. So now the classes are temporarily living in _sitemap.scss and can easily confuse the situation when these reusable classes need to be used on other pages. I will make sure to create the _page-cards.scss file asap, but in the meantime this is the quick fix!)

### PR DESCRIPTION
**Pull request by ItsLaszlo**
Fixes #1154 

Nested requested classes. Hopefully fixes the bleeding happening on issue #752

### CODE REVIEW

**Comment by daniellex0** [comment]
Hey @martham0 (FYI it's definitely possible that I am looking at the wrong file and you have done this already, or I am confusing this in some way, so take what I say with a grain of salt lol)-

But is it possible to move this section in the purple rectangle-

Screen Shot 2021-03-06 at 6 49 07 PM

... and nest it under this `.sitemap-page` class:

Screen Shot 2021-03-06 at 6 49 15 PM

 ... so that it only affects your page cards and not the ones on Travis' page?

File: _sass/components/_sitemap.scss [hunk 1]
```diff
@@ -40,92 +39,83 @@
...
+      a {
+        display: flex;
+        justify-content: center;
+      }
     }
   }
 }
 
-
 /*---------------------------
```
Comment (trtincher) [inline _sass/components/_sitemap.scss]: So everything below this still is not nested, if I am looking at this correctly. All of this should be nested as well.

File: _sass/components/_sitemap.scss [hunk 2]
```diff
@@ -149,21 +139,20 @@
...
-      padding: 32px 16px;
-      margin: 12px auto;
-  }
+    padding: 32px 16px;
+    margin: 12px auto;
   }
-
-
-  @media #{$bp-tablet-up}{
-    .page-card-lg:nth-child(1n+2){
```
Comment (trtincher) [inline _sass/components/_sitemap.scss:164]: As this is currently there will still be a bleeding problem I believe.

**Review by trtincher — CHANGES_REQUESTED** [review]

File: _sass/components/_sitemap.scss (hunk 1 above)
Comment (ItsLaszlo) [reply _sass/components/_sitemap.scss]: Fixed!

**Review by daniellex0 — APPROVED** [review]
I think this should work... but @trtincher you're the nesting expert - if you think this looks good, maybe we should merge this and see how it affects your page?

**Review by trtincher — APPROVED** [review]
@martham0 Yeah that will do it I think. Thanks!

**Pull request merged**